testassets/InteropTestsNativeWebsite/InteropServer.cs

```diff
@@ -0,0 +1,81 @@
+﻿#region Copyright notice and license
```

Comment:
nit: InteropTestsNativeWebsite -> InteropTestsNativeServer  (the Grpc.Core-based interop server is not a website).